docs/user_guide/geojson/geojson.md

```diff
@@ -237,6 +237,23 @@ colormap.add_to(m)
 m
 ```
 
+>
+> **Caveat**
```

Comment:
@Conengmo do you have any suggestion to make this more prominent, like a red warning sign?

Also, I'd like to link the `geopandas_and_geo_interface.html#Adding-style` to the GeoPandas object suggestion below but I have no idea how to do that here.

Reply:
I think it is clear enough with the specified Caveat. It is a very tricky bug, but I also imagine it is not a very common software pattern to create style functions in a loop.